prep-html-syntax/prep-html-syntax/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<body>
+
+<h1 style="color: brown;">My favorite Start Wars character is Ashoka!</h1>
+<img src="https://static.wikia.nocookie.net/starwars/images/1/12/AhsokaTano-AG.png/revision/latest?cb=20201130235153" alt="Ashoka">
+<p style="color: blue;">I really do not know</p>
+
+
+
+</body>
```

Comment:
Hi Santiago, be sure to divide your page into two specific sections. The `<head>` should open and close before the `<body>` opens and closes. The visible content of the page should go in the `<body>`.

```html
<!DOCTYPE html>
<html>
  <head>
    <!-- meta info like <title></title> goes in here -->
  </head>
  <body>
    <!-- visible content like <h1></h1> and <p></p> goes in here -->
  </body>
</html>
```

Reply:
<img width="956" alt="Screenshot (292)" src="https://user-images.githubusercontent.com/85325624/128101832-0d8627e1-159e-49a1-8042-c79088cbdf0d.png">

Hi Tim, I changed it. Should I submit again? If I have to, how do it? Thanks